Makefile

```diff
@@ -152,7 +152,7 @@ tar:
 	fakeroot debian/rules clean || make distclean
 	x=$$(basename $$(pwd)); \
 	cd ..; \
-	tar czf $$x.tar.gz --exclude=debian --exclude=.bzr* --exclude=.depend $$x
+	tar czf $$x.tar.gz --exclude=debian --exclude-vcs --exclude-vcs-ignores $$x
```

Comment:
The `--exclude-vcs` and `--exclude-vcs-ignores` are only in newer tar versions. Might want to research this a bit more.